docs/ballerina-by-example/examples/grpc-bidirectional-streaming/client/chat.pb.bal

```diff
@@ -1,59 +1,62 @@
 // This is an auto generated client stub which is used to communicate between gRPC client.
-package client;
-
 import ballerina/grpc;
 import ballerina/io;
 
 // Non-blocking client
-struct chatStub {
-    grpc:Client clientEndpoint;
-    grpc:ServiceStub serviceStub;
-}
+public type chatStub object {
+    public {
+        grpc:Client clientEndpoint;
+        grpc:ServiceStub serviceStub;
+    }
 
-function <chatStub stub> initStub (grpc:Client clientEndpoint) {
-    grpc:ServiceStub navStub = {};
-    navStub.initStub(clientEndpoint, "non-blocking", descriptorKey, descriptorMap);
-    stub.serviceStub = navStub;
-}
+    function initStub (grpc:Client clientEndpoint) {
+        grpc:ServiceStub navStub = new;
+        navStub.initStub(clientEndpoint, "non-blocking", descriptorKey, descriptorMap);
+        self.serviceStub = navStub;
+    }
 
-function <chatStub stub> chat (typedesc listener) returns (grpc:Client|error) {
-    var res = stub.serviceStub.streamingExecute("chat/chat", listener);
-    match res {
-        grpc:ConnectorError err => {
-            error e = {message:err.message};
-            return e;
-        }
-        grpc:Client con => {
-            return con;
+    function <chatStub stub> chat (typedesc listener) returns (grpc:Client|error) {
```

Comment:
You must remove <chatStub stub> from this function since it is part of the ChatStub object